chrysom/basicClient_test.go

```diff
@@ -147,6 +144,20 @@ func TestSendRequest(t *testing.T) {
 			MockError: nil,
 			MockAuth:  "basic xyz",
 		},
+		{
+			Description: "Happy path with default http client",
+			Method:      http.MethodPut,
+			URL:         "http://argus-hostname.io",
```

Comment:
Stay with something based on `example.com` because the url here could be a legitimate one & `example.com` is supposed to be reserved for testing.

Reply:
I didn't know that about `example.com`, that's neat!